server/test-data/checksum_sdk/create_sdk.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
+zip -r test_sdk.zip defoldsdk/
+printf '%s' "$(sha256 -q ./test_sdk.zip)" > test_sdk.sha256
```

Comment:
Used `printf` instead of `echo -n` because it's a portable way to write something to file without newline.